Serilog.TestLogger/SerilogTestLogger.cs

```diff
@@ -7,14 +7,14 @@
 // ReSharper disable once CheckNamespace
 namespace Serilog.TestLogger
 {
-    public class SerilogTestLogger : ILogger, ILogEventsContainer
+    public class SerilogTestLogger : ILogger, ILogEventsContainer, IDisposable
     {
         private readonly SerilogTestSink testSink = new SerilogTestSink();
-        private readonly ILogger innerLogger;
+        private readonly Logger innerLogger;
 
-        public SerilogTestLogger()
+        public SerilogTestLogger(Func<LoggerConfiguration,LoggerConfiguration> loggerConfiguration = default)
         {
-            innerLogger = new LoggerConfiguration()
+            innerLogger = (loggerConfiguration?.Invoke(new LoggerConfiguration()) ?? new LoggerConfiguration())
```

Comment:
Very minor, I would suggest extracting a variable here, maybe something like `initialLoggerConfiguration`, to avoid complicated expression in the fluent chain (and avoid awkward parenthesis)